src/Certes/Certes.csproj

```diff
@@ -45,8 +45,8 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
     <PackageReference Include="Portable.BouncyCastle" Version="1.8.1.2" />
-    <PackageReference Include="System.Net.Http" Version="4.3.2" Condition="'$(TargetFramework)' == 'net45'" />
-    <PackageReference Include="System.ValueTuple" Version="4.3.1" Condition="'$(TargetFramework)' == 'netstandard1.3' Or '$(TargetFramework)' == 'net45'" />
+    <PackageReference Include="System.Net.Http" Version="4.3.2" Condition="'$(TargetFramework)' == 'net45' Or '$(TargetFramework)' == 'net47'" />
+    <PackageReference Include="System.ValueTuple" Version="4.4.0" Condition="'$(TargetFramework)' == 'netstandard1.3' Or '$(TargetFramework)' == 'net45' Or '$(TargetFramework)' == 'net47'" />
```

Comment:
thanks mate, some questions:
Does net47 need to reference ValueTuple?
I think net47 will fall back to net45 automatically, and it seems the changes making net47 exactly same as net45, could you explain the different?